### ISSUE TITLE
'Wezel' object has no attribute 'set_menu'

### ISSUE THREAD

**Issue by JoaoPeriquito** [openmiblab/pckg-wezel#81]
When I call "wzl.set_menu(iBEAT_dev)" the following message appears: _'Wezel' object has no attribute 'set_menu'_ - I am wondering if the way to set custom menus has changed.

**Comment by plaresmedima** [comment]
Yes it has - I had corrected that in exec.py - does this appear somewhere
else?

Prof. Steven Sourbron, PhD
Chair in Medical Imaging Physics
University of Sheffield, UK
*https
<https://www.sheffield.ac.uk/medicine/people/iicd/steven-sourbron>://www.sheffield.ac.uk/medicine/people/iicd/steven-sourbron
<https://www.sheffield.ac.uk/medicine/people/iicd/steven-sourbron>*
*Sheffield abdominal imaging research page*
<https://www.sheffield.ac.uk/medicine/research/research-themes/medical-imaging/medical-imaging-research/abdominal-imaging>
*Open source initiative in perfusion imaging* <http://www.osipi.org>
*Magnetic resonance imaging biomarkers for chronic kidney disease
<https://renalmri.org/>*

**Comment by JoaoPeriquito** [comment]
Everything seems to be fine if I use the default menu from wezel. I was
just trying to use wezel together with our ibeat menu by doing an editable
install and executing it.

**Comment by plaresmedima** [comment]
Yes have you tried using the exec.py function which is in iBEAt?

**Comment by JoaoPeriquito** [comment]
For testing wezel alone, I was using wezel exec.py all - all fine
.
For testing wezel with  iBEAt  menu, I was using iBEAt exec.py - from where
I got the described error

**Comment by plaresmedima** [comment]
are you checking the pull request? I assigned you as reviewer

https://github.com/QIB-Sheffield/iBEAt/pull/21

Prof. Steven Sourbron, PhD
Chair in Medical Imaging Physics
University of Sheffield, UK
*https
<https://www.sheffield.ac.uk/medicine/people/iicd/steven-sourbron>://www.sheffield.ac.uk/medicine/people/iicd/steven-sourbron
<https://www.sheffield.ac.uk/medicine/people/iicd/steven-sourbron>*
*Sheffield abdominal imaging research page*
<https://www.sheffield.ac.uk/medicine/research/research-themes/medical-imaging/medical-imaging-research/abdominal-imaging>
*Open source initiative in perfusion imaging* <http://www.osipi.org>
*Magnetic resonance imaging biomarkers for chronic kidney disease
<https://renalmri.org/>*
